src/components/SecondaryPanes/Expressions.js

```diff
@@ -52,6 +50,8 @@ const Expressions = React.createClass({
     loadedObjects: ImPropTypes.map.isRequired
   },
 
+  _input: (null: any),
```

Comment:
I moved it here to fix
<img width="588" alt="screen shot 2017-02-18 at 10 21 41 am" src="https://cloud.githubusercontent.com/assets/4562118/23089516/de118ff6-f5c4-11e6-9acf-d36720524dc8.png">